packages/manager/modules/pci/src/components/project/flavors-list/flavors-list.html

```diff
@@ -141,14 +141,21 @@
                             </div>
                         </oui-select-picker-section>
                         <oui-select-picker-section
-                            data-ng-if="$ctrl.region && !$ctrl.region.hasEnoughQuotaForFlavor(flavor)"
+                            data-ng-if="$ctrl.region && $ctrl.region.hasEnoughQuotaForFlavor(flavor)"
```

Comment:
@radireddy you set the condition to "if region has enough quota for flavor", but below, if we take a look at the translation used, we have "Your project quota doesn't allow you to add this type of machine in this region" (pci_project_flavors_quota_info). Is it normal?

Reply:
no its not, thanks for bringing it to my notice. I will change it.